web/public/js/services/cd-users-service.js

```diff
@@ -42,6 +42,13 @@ function cdUsersService(cdApi, $q){
         cdApi.post('profiles/youth/create', {profile: profile}, win, fail || topfail);
       }
     },
+    saveYouthProfilePromise: function(profile) {
```

Comment:
Not sure if I don't prefer a separation between both, let's talk abt it tomorrow